libs/steps.js

```diff
@@ -59,6 +59,22 @@ var steps = {
             oldVersion: versionProvider.readVersion().format()
         };
     },
+    scripts: function(config, prefix) {
+        var fileName = config.versionProvider.filePath,
+            promise;
+        if (fileName === 'package.json') {
+            var prefixMsg = prefix[0].toUpperCase() + prefix.substring(1),
+                cmd = JSON.parse(cat(fileName)).scripts[prefix+'releasy'],
```

Comment:
Se esse for o caso o pickVersionProvider já teria pego esse erro antes, já que ele tá um test -e no fileName.